test/src/themes/expression/expression_parser_test.dart

```diff
@@ -383,6 +388,17 @@ void main() {
       ], 'match(get(another-string),[literal(no-match-value),literal(a-string-value)],[literal(another-no-match-value)],literal(false),literal(false),literal(true))',
           true);
     });
+
+    test('parses an is-supported-script expression for latin script strings', () {
+      assertExpression(['is-supported-script', ['get', 'a-string']], 'isSupportedScript(get(a-string))', true);
```

Comment:
Great to have some tests! It would be good to have some `false` test cases too.

Reply:
The comment in `is_supported_script_expression.dart` addresses this, but apologies if that was not clear. At the moment, there are no cases for which this expression would be false, thanks to how rendering is implemented in the vector renderer / flutter map :)